tests/setup.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { test } from "@playwright/test";
+
+test("happy path setup", async ({ page }) => {
```

Comment:
Would rather call this something different

Reply:
Open for suggestions :)